docs/client_reference.rst

```diff
@@ -2077,6 +2077,18 @@ All exceptions are available as members of *aiohttp* module.
 
       Invalid URL, :class:`yarl.URL` instance.
 
+    .. attribute:: description
+
+      Invalid URL description, :class:`str` instance.
+
+.. exception:: InvalidRedirectUrl
+
+   Redirect URL used for fetching is malformed, e.g. it does not contain host
```

Comment:
It'd say it's the URL in the server's location header, not used for fetching.

Reply:
updated